### ISSUE TITLE
Locked elements should be activatable by default

### ISSUE THREAD

**Issue by rakina** [WICG/display-locking#80]
This was brought up in the TPAC CSSWG session last week. Non-activatability should be an intentional thing (because it means things on the page will be skipped by focus, a11y, etc) so it makes sense for locked elements to be activatable by default.

So `rendersubtree="invisible"` means it's invisible & activatable.
What should be the term for non-activatability?
Also, what does this mean for combinations with `holdupgrades` and `holdloads`?

**Comment by vmpstr** [comment]
What was the invisibleDOM equivalent of this? I seem to think that `static` was a keyword that would prevent activatability, or was that to prevent upgrades?

**Comment by rakina** [comment]
That was for preventing upgrades, yeah.
